### ISSUE TITLE
Investigate the effect of using batch normalization and layer normalization in DNN module

### ISSUE THREAD

**Issue by dbeinhauer** [dbeinhauer/mcs-source#48]
The DNN module might perform better while using batch or layer normalization after each linear layer application.

**Comment by dbeinhauer** [comment]
It looks that for batch normalization we have too small batches and layer. Layer normalization turns out to be the best option. We get slightly better results in comparison to variant without any normalization. In addition, the noisiness of the predictions (and training losses) is better too.
